learn-more.html

```diff
@@ -1,13 +1,200 @@
 <!DOCTYPE HTML>
 <html>
-  <head>
-    <title>Learn More</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<head>
+  <title>Motherflippin Homepage</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link href="style.css" rel="stylesheet" />
+</head>
+
+
+<body id="container">
+
+
+  <!-- header section -->
+  <div class="item header">
+    <a href="index.html">
+      <h1>
+        L E A R N &nbsp M O R E
+      </h1>
+    </a>
+    <div class="subhead">
+      SO, YOU'RE HERE TO LEARN MORE...
+
+      <div class=headrln>
+        <br> <br />
+        <hr>
+        <hr />
+      </div>
+    </div>
+
+  </div>
+
+
+
+  <!-- features secion w/ images-->
+
+
+  <section class="item features">
+    <div class="featsect">
+      <h4 style="text-align: center;"> Forgot what this page was supposed to look like (I didn't see a picture) but will
+        redo it once I figure it out...

```

Comment:
https://bootcamp.burlingtoncodeacademy.com/images/left-1.png

Here is the link to what the "Learn More" page should be.